glean-core/src/internal_metrics.rs

```diff
@@ -85,6 +101,15 @@ impl CoreMetrics {
                 }),
                 None,
             ),
+
+            invalid_timezone_offset: CounterMetric::new(CommonMetricData {
+                name: "invalid_timezone_offset".into(),
+                category: "glean.time".into(),
+                send_in_pings: vec!["metrics".into()],
+                lifetime: Lifetime::Ping,
+                disabled: false,
```

Comment:
What, no clever `#[cfg(build_time > '2021-06-30T00:00:00+00:00')]` cargo hack?

Reply:
I _did_ think about it to be honest, but then I remembered that I should get this finished _today_